string name
float32 longitude
float32 latitude
int32 fid
